src/pudl_rmi/coordinate.py

```diff
@@ -0,0 +1,247 @@
+
+        Args:
+            clobber (boolean): True if you want to regenerate the connection
+                between the depreciation data and EIA whether or not the output
+                is already pickled. Default is False.
+            clobber_ppl (boolean): True if you want to regenerate the master
+                unit list whether or not the output is already pickled. Default
+                is False.
+            clobber_d (boolean): True if you want to regenerate the
+                depreciation data whether or not the output is already pickled.
+                Default is False.
+        """
+        clobber_any = any([clobber, clobber_ppl, clobber_d])
+        if not pudl_rmi.FILE_PATH_DEPRISH_EIA.is_file() or clobber_any:
+            deprish_match_df = pudl_rmi.connect_deprish_to_eia.execute(
+                plant_parts_df=self.get_plant_part_list(clobber=clobber_ppl),
+                # deprish_df=self.get_deprish(clobber=clobber_d),
+            )
+            deprish_match_df.to_pickle(pudl_rmi.FILE_PATH_DEPRISH_EIA)
+        else:
+            deprish_match_df = pd.read_pickle(pudl_rmi.FILE_PATH_DEPRISH_EIA)
+        return deprish_match_df
+
+    def get_ferc1_to_eia(self, clobber=False, clobber_ppl=False):
+        """
+        Generate or grab a connection between FERC1 and EIA.
+
+        Either generate or grab an on-disk cached version of the connection
+        between the FERC1 plant data and the EIA plant part list.
+
+        Args:
+            clobber (boolean): True if you want to regenerate the master unit
+                list whether or not it is saved at the file_path_mul
+            clobber_ppl (boolean): Generate and cache a new interim
+                output of the EIA plant part list and generate a new version of
+                the depreciaiton to FERC1 output. Default is False.
+        """
+        # if any of the clobbers are on, we want to regenerate the main output
+        clobber_any = any([clobber, clobber_ppl])
+        if not pudl_rmi.FILE_PATH_FERC1_EIA.is_file() or clobber_any:
+            logger.info(
+                "FERC to EIA granular connection not found at "
+                f"{pudl_rmi.FILE_PATH_FERC1_EIA}... Generating a new output."
+            )
+            connects_ferc1_eia = pudl_rmi.connect_ferc1_to_eia.execute(
+                self.pudl_out,
+                self.get_plant_part_list(clobber=clobber_ppl)
+            )
+            # export
+            connects_ferc1_eia.to_pickle(pudl_rmi.FILE_PATH_FERC1_EIA)
+        else:
+            logger.info(
+                "Reading the FERC1 to EIA connection from "
+                f"{pudl_rmi.FILE_PATH_FERC1_EIA}"
+            )
+            connects_ferc1_eia = pd.read_pickle(pudl_rmi.FILE_PATH_FERC1_EIA)
+        return connects_ferc1_eia
+
+    def get_deprish_to_ferc1(
+        self,
+        clobber=False,
+        clobber_ppl=False,
+        clobber_de=False,
```

Comment:
Here you're using `clobber_de` instead of `clobber_d` as you did above. Let's standardize on a single code for all of these things, and I think longer and more readable is better. `clobber_deprish` `clobber_ferc1_eia` etc.

Reply:
Oh whoops, I see this means something different here, `deprish_eia`  Longer names make it easier to differentiate things which are actually different too.